skia/RNGoogleCastModule.h

```diff
@@ -16,11 +16,11 @@ namespace facebook {
 namespace xplat {
 
 class RNGoogleCastModule : public module::CxxModule {
- public:
-  RNGoogleCastModule();
-  std::map<std::string, folly::dynamic> getConstants() override;
-  std::vector<Method> getMethods() override;
-  std::string getName() override;
+  public:
```

Comment:
General standard followed is one space for access specifier and furter one space for data member, not two. Can you correct it.
The earlier one is right.